lsp-methods.el

```diff
@@ -1404,11 +1404,15 @@ https://microsoft.github.io/language-server-protocol/specification#textDocument_
 (defun lsp--annotate (item)
   (let* ((table (plist-get (text-properties-at 0 item) 'lsp-completion-item))
          (detail (gethash "detail" table nil))
-         (kind (aref lsp--completion-item-kind (gethash "kind" table nil))))
-    (concat
-     " "
-     detail
-     (when kind (format " (%s)" kind)))))
+         (kind-index (gethash "kind" table nil)))
+    ;; We need check index before call `aref'.
+    (when kind-index
+      (setq kind (aref (remove nil lsp--completion-item-kind) kind-index))
```

Comment:
`lsp--completion-item-kind` is an array. `(remove nil ...)` will make indexes off-by-1. Do you have a better solution?